### ISSUE TITLE
Add stamina recovery delay to normal stamina

### ISSUE THREAD

**Issue by Tenuuu** [murphy-slaw/staminafortweakers#18]
Would there be a possibility to add in the stamina recovery delay to stamina as a whole and not just when exhaustion occurs?

**Comment by murphy-slaw** [comment]
Yeah, that makes sense as an option. I will probably implement it as a separate timer so that it's possible to set a longer recovery pause on actual exhaustion.

**Comment by murphy-slaw** [comment]
Released in 1.3.0.
